### ISSUE TITLE
Cannot install with CocoaPods iOS

### ISSUE THREAD

**Issue by felixchan** [ivpusic/react-native-image-crop-picker#1003]
- react-native-image-crop-picker v0.24.0
- react-native v0.59
- iOS

After installing into node_modules, I modified Podfile per the instructions:

```
# Uncomment the next line to define a global platform for your project
platform :ios, '9.0'

target 'atom' do
  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
  # use_frameworks!

  pod 'Firebase/Core', '~> 5.20.1'
  pod 'Firebase/Firestore', '~> 5.20.1'
  pod 'Firebase/Auth', '~> 5.20.1'
  pod 'Firebase/Storage', '~> 5.20.1'
  pod 'Firebase/Database', '~> 5.20.1'
  pod 'Firebase/Messaging', '~> 5.20.1'
  pod 'Firebase/Functions', '~> 5.20.1'
  pod 'Firebase/DynamicLinks', '~> 5.20.1'
  pod 'Firebase/Performance', '~> 5.20.1'


  # this is very important to have!
  rn_path = '../node_modules/react-native'
  pod 'yoga', path: "#{rn_path}/ReactCommon/yoga/yoga.podspec"
  pod 'React', path: rn_path, subspecs: [
  'Core',
  'RCTActionSheet',
  'RCTAnimation',
  'RCTGeolocation',
  'RCTImage',
  'RCTLinkingIOS',
  'RCTNetwork',
  'RCTSettings',
  'RCTText',
  'RCTVibration',
  'RCTWebSocket'
  ]
  pod 'RNImageCropPicker', :path =>  '../node_modules/react-native-image-crop-picker'
  # pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler'
end

 post_install do |installer|
   installer.pods_project.targets.each do |target|
     if target.name == "React"
       target.remove_from_project
     end
   end
 end
```

When running `pod install`, I get:

```
$ pod install
Ignoring unf_ext-0.0.7.4 because its extensions are not built.  Try: gem pristine unf_ext --version 0.0.7.4
Analyzing dependencies
Fetching podspec for `RNImageCropPicker` from `../node_modules/react-native-image-crop-picker`
Fetching podspec for `React` from `../node_modules/react-native`
Fetching podspec for `yoga` from `../node_modules/react-native/ReactCommon/yoga/yoga.podspec`
[!] CocoaPods could not find compatible versions for pod "React/Core":
  In snapshot (Podfile.lock):
    React/Core

  In Podfile:
    React/Core (from `../node_modules/react-native`)

None of your spec sources contain a spec satisfying the dependencies: `React/Core (from `../node_modules/react-native`), React/Core`.

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

Note: as of CocoaPods 1.0, `pod repo update` does not happen on `pod install` by default.
```

**Comment by felixchan** [comment]
When I remove the React + subspecs, the pod will install successfully.

However, the build will not succeed.

**Comment by Purvik** [comment]
`post_install do |installer|
   installer.pods_project.targets.each do |target|
     if target.name == "React"
       target.remove_from_project
     end
   end
 end`

Remove this one and try `pod install` again.

**Comment by intmainvoid** [comment]
I experienced the same problem and fixed it by changing my platform to 12.0 like so:
```
platform :ios, '12.0'
```
Issue https://github.com/ivpusic/react-native-image-crop-picker/issues/762 seems related

**Comment by txinhua** [comment]
@Felix Chan  you should modify  the podspec s.dependency 'React-Core'
  s.dependency 'React-RCTImage' to s.dependency 'React/Core'
